ICAFIX/hcp_fix_multi_run

```diff
@@ -379,116 +385,352 @@ log_Msg "PWD: $DIR"
 # AND since 'addpath' *prepends* to the matlab path, the version in HCPCIFTIRWDIR will therefore take precedence.
 ML_PATHS="addpath('${HCPPIPEDIR}/global/fsl/etc/matlab'); addpath('${HCPCIFTIRWDIR}'); addpath('${HCPPIPEDIR}/global/matlab/icaDim'); addpath('${HCPPIPEDIR}/global/matlab'); addpath('${this_script_dir}/scripts');"
 
-## ---------------------------------------------------------------------------
-## Preparation (highpass) on the individual runs
-## ---------------------------------------------------------------------------
 
-#Loops over the runs and do highpass on each of them
-log_Msg "Looping over files and doing highpass to each of them"
-
-NIFTIvolMergeARRAY=()
-NIFTIvolhpVNMergeARRAY=()
-SBRefVolARRAY=()
-MeanVolARRAY=()
-VNVolARRAY=()
-CIFTIMergeARRAY=()
-CIFTIhpVNMergeARRAY=()
-MeanCIFTIARRAY=()
-VNCIFTIARRAY=()
-MovementNIFTIMergeARRAY=()
-MovementNIFTIhpMergeARRAY=()
-MovementTXTMergeARRAY=()
+## ===========================================================================
+## Skip all steps until FIX if RunFIXOnly is set to TRUE
+## ===========================================================================
+
+if [[ "${RunFIXOnly}" == "FALSE" ]]; then
```

Comment:
Please use `opts_StringToBool` for sanity checking, case insensitivity, and user friendliness, see here:

https://github.com/Washington-University/HCPpipelines/blob/9cfa37a378dff26fd5aa4fd2aad24d0136830f0f/global/scripts/RSNregression.sh#L72
https://github.com/Washington-University/HCPpipelines/blob/9cfa37a378dff26fd5aa4fd2aad24d0136830f0f/global/scripts/RSNregression.sh#L211

Reply:
Ok. `opts_StringToBool` is used in the upcoming new PR.